CHANGELOG.md

```diff
@@ -0,0 +1,102 @@
+# FinOps Open Cost and Usage Specification Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+**Added:**
+
+- `RegionId` column
+
+**Changed:**
+
+- `Column naming convention` attribute includes the following new provisions:
+  - Column IDs should not exceed 50 characters.
+- `EffectiveCost` column updated to clarify it does not mix or "blend" costs across multiple charges.
```

Comment:
Periods in some bullets and not in others is bothering me. How would others feel if the main bullet is the specific asset we're updating and the change description was always in a nested bullet? That is needed when there are multiple changes but we could apply it consistently for single changes as well. Thoughts?
```suggestion
- `EffectiveCost` column
  - Updated to clarify it does not mix or "blend" costs across multiple charges.
```